legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/HelperValueSpecificationBuilder.java

```diff
@@ -207,8 +224,8 @@ else if (foundProperty instanceof QualifiedProperty)
                 List<Variable> lambdaParams = new FastList<>();
                 lambdaParams.add(automaLambdaparam);
                 automapLambda.parameters = lambdaParams;
-                MilestoningDatePropagationHelper.updateMilestoningPropagationContextForAutoMap(foundProperty, context, appliedProperty.parameters.size(), processedParameters.get(0), processingContext);
```

Comment:
Is this method use anywhere else?  I did not find any usage, hence if that is correct, lets clean up the code of unused methods.

Reply:
done